config/manifests/bases/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -489,5 +489,5 @@ spec:
   provider:
     name: Dell Technologies
     url: https://github.com/dell/csm-operator
-  replaces: dell-csm-operator.v1.0.0
-  version: 1.1.0
+  replaces: dell-csm-operator.v1.2.0
```

Comment:
Value for Replaces should be n-1 version.. It should 1.1.0. Please do this correction in both the CSV's

Reply:
taken care